mobile/src/shared/components/Layout.tsx

```diff
@@ -1,12 +1,79 @@
-import type { FunctionComponent, ReactElement } from "react";
-import { Outlet } from "react-router-dom";
+import { useEffect, useState } from "react";
+import { Link as RouterLink, Outlet } from "react-router-dom";
+
+import {
+  AppBar,
+  Box,
+  Container,
+  Link,
+  MenuItem,
+  Toolbar,
+  Typography,
+} from "@mui/material";
+
+import { NullableUser } from "../SharedTypes";
+
+import { useUserRepository } from "../SharedHooks";
+
+const Layout = () => {
+  const userRepository = useUserRepository();
+  const [user, setUser] = useState<NullableUser>(null);
+
+  useEffect(() => {
+    userRepository.me().then((me) => {
+      setUser(me);
+    });
+  }, []);
 
-const Layout: FunctionComponent<{}> = (): ReactElement => {
   return (
     <>
-      <header></header>
+      <AppBar>
+        <Container maxWidth="xl">
+          <Toolbar disableGutters>
+            <Box sx={{ flexGrow: 1 }}>
```

Comment:
```suggestion
            <Box flexGrow={1}>
```

All the `sx` keys are accessible as props on `Box`, and that's more readable.